config/initializers/0_application_config.rb

```diff
@@ -5,7 +5,7 @@ def self.[](key)
     if ENV.key?(key)
       ENV[key]
     else
-      Rails.logger.debug("Unset ENV variable: #{key}.")
+      Rails.logger.debug { "Unset ENV variable: #{key}." }
```

Comment:
ah - this explains the failure in spec/initializers/application_config_spec.rb:7 - changing from method args to a block here made the expected "receive with an arg" fail.

Reply:
thanks for catching this!